src/sentry/integrations/utils/metrics.py

```diff
@@ -173,7 +173,9 @@ def record_success(self) -> None:
         self._terminate(EventLifecycleOutcome.SUCCESS)
 
     def record_failure(
-        self, failure_reason: BaseException | str | None = None, extra: dict[str, Any] | None = None
+        self,
+        failure_reason: BaseException | str | None = None,
+        extra: Mapping[str, Any] | None = None,
```

Comment:
Are these just added by a formatter?

Reply:
No. It's a pass-by change since it's best practice to use `Mapping` instead of `dict` in function signature.